src/orion/core/cli/db/load.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+# pylint: disable=,protected-access
+"""
+Storage import tool
+===================
+
+Import database content from a file.
+
+"""
+import logging
+
+from orion.core.cli import base as cli
+from orion.core.io import experiment_builder
+from orion.core.worker.storage_backup import load_database
+from orion.storage.base import setup_storage
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+DESCRIPTION = "Import storage"
+
+
+def add_subparser(parser):
+    """Add the subparser that needs to be used for this command"""
+    load_parser = parser.add_parser("load", help=DESCRIPTION, description=DESCRIPTION)
+
+    cli.get_basic_args_group(load_parser)
+
+    load_parser.add_argument(
+        "file",
+        type=str,
+        help="File to import",
+    )
+
+    load_parser.add_argument(
+        "-r",
+        "--resolve",
+        type=str,
+        choices=("ignore", "overwrite", "bump"),
+        required=True,
```

Comment:
If there are no conflicts it would be great if the command could work without specifying the resolution method. Did we ever discuss about this? I only have vague memories about it... :thinking:

Reply:
Done!